wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
-### Use of IP Addresses
+All communications between and among Clients, Servers, Directories, and third-party protocol users are sent without encryption.
 
-When you connect to Server, the Server operator can see your IP address while you are connected.  If the Server operator has enabled logging (which is off by default) your IP address will also be logged and stored in the Server's log file.
+## Jamulus Servers
 
-The IP addresses of all Servers registered with the Directory can also be seen by third parties for informational or other purposes (for example [here](https://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
+When you connect to a Server, either directly or through a Directory, the operator of that Server is responsible for its operation policy, privacy policy, and data use policy. While unmodified Servers do not log or store your connection or profile information, some modified Servers may do this; therefore, you should have no expectation of privacy with respect to your profile information or internet address.
 
-### Audio Recordings
+## Chat Exchanges
 
-You will see a notice if you are connected to a Jamulus Server when Server recording is turned on. Recordings of each player are stored by the Server separately as .WAV files and only the Server operator has access to them unless they choose to make them available to third parties.
+Chats are textual messages that can be exchanged between Clients connected to the same Server. Everyone connected to a Server can see all chats and there should be no expectation of privacy with respect to information sent through the chat feature of Jamulus. While unmodified Servers do not log or store chats, some modified Servers may do this.
 
-### Text Chat
+## Audio Recordings
 
-When you type a message in the Chat Window, other connected players can see that, but chats are not stored by the Server and neither the Server operator nor any third parties have access to them.
+Unmodified Servers will display a notice if recording is turned on. Recordings of each player are stored by the Server and are controlled by the Server operator.
 
-### Data Transmission
+It is possible for connected Clients to make recordings of sessions outside of Jamulus itself, for which there may be no notice or indication. Jamulus has no way to detect or control these situations and makes no representations as to the collection or use of such recording data.
 
-Please note that all audio and text data is sent and received between the Jamulus Server and Client without encryption.
+## Directories
+
+Directories are central connection points for locating Servers. The Jamulus Client comes with a pre-defined list of Directories for the convenience of users. These Directories are independently operated but are represented as running unmodified versions of the Jamulus software.
```

Comment:
```suggestion
Directories are central connection points for locating Servers. The Jamulus Client comes with a pre-defined list of Directories for the convenience of users. These Directories are independently operated but are represented as running unmodified versions of the Jamulus software. If you do not wish to send your information to the Directories, you will need to connect to Servers directly.
```

Reply:
Not entirely true.  Volker runs latest release on his built in directories.  I run the latest tagged release on the other Genre servers, which will edge towards the next release as we get closer.  (I also run the latest release on private.jamulus.io (which is the same as updatecheck2) and master on my other directories.)

Reply:
Sorry, can you explain what you mean about what's not entirely true?

Reply:
I should have commented directly on the change, sorry - I'd not seen it when reviewing.  I wasn't commenting on your suggestion.  The issue is that the directories are run by people compiling any version of Jamulus with any changes made to the source that they want to incorporate.  Stating that these are "unmodified versions of the Jamulus software" is probably stretching the truth beyond breaking point.

Reply:
So it would be true to say "These Directories are operated by members of the Jamulus team and run unmodified versions of the Jamulus software." ?

Reply:
That still doesn't say what an "unmodified version of the Jamulus software" is.  Is it the latest released .deb?  Is it the version on with some commit id?

In fact, it's not necessarily either of those as I think I passed the patch file directly to Volker for the bug fix and said "Here, patch your source until it's in the tree".

So no one can say what version of Jamulus the built in directories are running.

Reply:
OK, but the central point is that in connecting to any directory you are connecting to a system that isn't under your control and could harvest information or otherwise act against you. 

So might we just say "These Directories are independently operated instances of Jamulus is Directory mode." and leave the reader to make their own judgement about that in the same way that (presumably) they would in connecting their web browser to somebody else's website?